new_platforms/src/enc/optee/report_optee.c

```diff
@@ -59,23 +59,33 @@ oe_result_t oe_verify_report(
     }
 
     // validate the chain is properly rooted
+    mbedtls_x509_crt* root = &chain;
+    while (root->next)
+        root = root->next;
```

Comment:
Agreed. I think I will take that to the next level and create a spec for the attestation to begin with. I think it would be good to get a consensus on this. Both local and remote.